core/src/main/java/org/jboss/pnc/rex/core/config/api/HttpConfiguration.java

```diff
@@ -56,15 +55,8 @@ public interface HttpConfiguration {
      *
      * @return FT configuration for unexpected failures
      */
-    HttpRetryPolicy requestRetryPolicy();
+    RequestRetryPolicy requestRetryPolicy();
+
+    StatusCodeRetryPolicy statusCodeRetryPolicy();
```

Comment:
A simple JavaDoc what this does.

Reply:
bump

Reply:
There is one https://github.com/matejonnet/rex/blob/http-retry/core/src/main/java/org/jboss/pnc/rex/core/config/api/HttpConfiguration.java

Reply:
Sorry about this. I've expected that GitHub will mark the file as outdated but it didn't for some reason